rust/catalyst-types/src/id_uri/mod.rs

```diff
@@ -449,17 +449,17 @@ impl IdUri {
     /// # Examples
     ///
     /// ```rust
-    /// use catalyst_types::id_uri::{key_rotation::KeyRotation, role_index::RoleIndex, IdUri};
+    /// use catalyst_types::id_uri::{key_rotation::KeyRotation, role_index::RoleId, IdUri};
     ///
     /// let id_uri =
-    ///     "id.catalyst://user:1735689600@cardano/FftxFnOrj2qmTuB2oZG2v0YEWJfKvQ9Gg8AgNAhDsKE/7/5"
+    ///     "id.catalyst://user:1735689600@cardano/FftxFnOrj2qmTuB2oZG2v0YEWJfKvQ9Gg8AgNAhDsKE/3/5"
```

Comment:
7 should be OK.  It might be an unknown role id, but it shouldn't fail to de-serialize.